nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/TransformXml.java

```diff
@@ -101,16 +101,16 @@
     public static final PropertyDescriptor CACHE_SIZE = new PropertyDescriptor.Builder()
             .name("cache-size")
             .displayName("Cache size")
-            .description("Maximum size of the stylesheet cache.")
+            .description("Maximum number of stylesheets to cache. Zero disables the cache.")
             .required(true)
             .defaultValue("100")
```

Comment:
This seems super high as a default number of stylesheets. I don't have much experience with xml and stylesheets but I would think that the average user of a default value wouldn't need to cache use more than 10. I will defer to your experience with the xml and stylesheets if you think otherwise though.